com.unity.netcode.gameobjects/Tests/Runtime/NetworkBehaviourUpdaterTests.cs

```diff
@@ -143,14 +158,14 @@ void AddNetworkBehaviour(Type type, GameObject prefab)
                             var varInt = networkVariable as NetworkVariable<int>;
                             var varUpdateResult = new MultiInstanceHelpers.CoroutineResultWrapper<bool>();
                             yield return MultiInstanceHelpers.WaitForCondition(() => varInt.Value == updatedValue, varUpdateResult);
-                            Assert.That(varUpdateResult.Result, Is.True);
+                            Assert.That(varUpdateResult.Result, Is.True, "Variable should be updated on the clients");
```

Comment:
nit: To help even more, could it be worth it to log on which NetworkBehaviour this failed? This way we know whether a behaviour with 1 or 2 netvars has triggered this error.